pybossa/api/__init__.py

```diff
@@ -847,3 +847,75 @@ def user_has_partial_answer(short_name=None):
     task_id_map = get_user_saved_partial_tasks(sentinel, project_id, current_user.id, task_repo)
     response = {"has_answer": bool(task_id_map)}
     return Response(json.dumps(response), status=200, mimetype="application/json")
+
+
+@jsonpify
+@csrf.exempt
+@blueprint.route('/llm', defaults={'model_name': None}, methods=['POST'])
+@blueprint.route('/llm/<model_name>', methods=['POST'])
+@ratelimit(limit=ratelimits.get('LIMIT'), per=ratelimits.get('PER'))
+def large_language_model(model_name):
+    """Large language model endpoint
+    The POST data format can be:
+    data = {
+        "instances": [
+            {
+                "context": "Identify the company name: Microsoft will release Windows 20 next year.",
+                "temperature": 1.0,
+                "seed": 12345,
+                "repetition_penalty": 1.05,
+                "num_beams": 1,
+            }
+        ]
+    }
+    or
+    data = {
+        "prompts": "Identify the company name: Microsoft will release Windows 20 next year."
+    }
+    """
+    if model_name is None:
+        model_name = 'flan-ul2'
+    endpoints = current_app.config.get('LLM_ENDPOINTS')
+    model_endpoint = endpoints.get(model_name.lower())
+
+    if not model_endpoint:
+        return abort(403, f'Model with name {model_name} does not exist.')
```

Comment:
With this configuration not present, it would be LLM unsupported. Hence, should it be instead
``` abort(400, "LLM is unsupported on this platform")```